NOTICE

```diff
@@ -1,15 +1,13 @@
-/**
- * Copyright 2021 Bonitasoft S.A.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+Copyright 2021-present Bonitasoft S.A.
```

Comment:
> 2021-present

Is `present` a valid value?

Reply:
We decided to use this wording long time ago: https://github.com/process-analytics/bpmn-visualization-js/pull/1646/files
But if you have something to propose, go on 😸

Reply:
My concern was more "Is this **legally** acceptable to consider that every time the present year changes (every 01/01 of any year), the copyright is "automatically" reconducted ?"

Reply:
Well, I do not know. If this is an issue, I can revert this part.
My objective in this PR was to remove the extra TS/JS comments, the copyright change was just a bonus

Reply:
I have reverted this change in 36450fb